### PR TITLE
Remove Log streaming from showBake page

### PR DESCRIPTION
**Pull request by michaelwmcnamara**
## What does this change?
This PR is preparation for upgrading to Scala 2.13
Amigo uses  the Play "iteratees" and "reactive-streams" libraries to surface the logs in real time on the  "show-bake" pages for a bake that is in progress.
These libraries have reached their end of life and do not have a version supported in Scala 2.13.
In later versions of Play, Akka streams are used in place of the reactive streams used to-date. However the implementation of these is different enough to be non-trivial.
It was also determined that live-streaming the logs for a bake that can often take 20 minutes to complete was not of great value as it is unlikely to be watched for that long. The logs are loaded when the page is refreshed after completion of bake - which is  when a user is most likely to be looking at them in any case.
As such it was determined that - in order to allow an upgrade of Amigo to scala 2.13 - it would be necessary to remove the live streaming of logs functionality.

## How to test

Tested by compiling locally,
Running tests locally
Deploying to CODE
Creating a new Bake on the code Amigo site
When refreshed after completion of Bake confirm that:
- [x] Bake completed successfully
- [x] The auto-scroll dropdown was not shown on the show-bake page;
- [x] Logs were displayed in the correct section on the show-bake page; 
- [x] When package changes were forced on an existing recipe, those changes were displayed in the Package Changes section of the show-bake page;
- [x] All installed packages were listed in the "All installed packages" section of the show-bake page.

Before change - streaming happens during bake:
before

After change - no streaming during bake:
afterstart2

After change - When refreshing after bake completion logs are present:
afterstart3

### CODE REVIEW

**Review by akash1810 — COMMENTED** [review]
Could we add a before and after screenshot/video too please?

**Comment by jfsoul** [comment]
It might be nice to add a message (if we can) to the initial log window after you hit bake. Something like `Refresh the page to view new log messages`?

**Comment by jfsoul** [comment]
Is there also some code [here](https://github.com/guardian/amigo/blob/6171d827cf13e649b7e940c5c9957d5b58929701/public/javascripts/show-bake.js#L29) we could remove? (I haven't checked thoroughly)

File: app/views/showBake.scala.html [hunk 1]
```diff
@@ -43,16 +43,11 @@ <h1><a href="@routes.RecipeController.showRecipe(bake.recipe.id)">@bake.recipe.i
...
-    <label>
-      <input id="tail-log" type="checkbox" value="" checked>
-      Auto scroll
-    </label>
-  </div>
-
-  <div class="well" id="packer-output" data-eventsource-url="@controllers.routes.BakeController.bakeEvents(bake.recipe.id, bake.buildNumber)" data-initial-highest-log-number="@{bakeLogs.lastOption.map(_.logNumber).getOrElse(-1)}">
-    @for(log <- bakeLogs) {
+  <div class="well" id="packer-output">
+    @if(bakeLogs.isEmpty){<div class="bake-log">Refresh the page to view new log messages</div>}else{
```
[the remark this answers is not in the document]
Comment (michaelwmcnamara) [reply app/views/showBake.scala.html]: Simpler is better! Removed the conditional now.

**Comment by michaelwmcnamara** [comment]
> Is there also some code [here](https://github.com/guardian/amigo/blob/6171d827cf13e649b7e940c5c9957d5b58929701/public/javascripts/show-bake.js#L29) we could remove? (I haven't checked thoroughly)

There is - also removed some libraries that are not / are no longer in use.

File: public/stylesheets/main.css [hunk 2]
```diff
@@ -64,6 +64,10 @@ body { padding-top: 70px; }
     margin-left: auto;
 }
 
+.bake--info--refresh--message {
+    align-items: center;
```
Comment (jfsoul) [inline public/stylesheets/main.css:68]: Is this working as you intended? It doesn't seem to be center aligned in the screenshots. I'm also not sure I understand the naming convention of classes here - what does double dash represent?

**Review by jfsoul — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -2,9 +2,7 @@ name: CI
 on:
   pull_request:
   workflow_dispatch:
-  push:
-    branches:
-      - main
```
[the remark this answers is not in the document]
Comment (michaelwmcnamara) [reply .github/workflows/ci.yaml:7]: indeed - putting it back now.

**Pull request merged**